plugins/outputs/whatap/README.md

```diff
@@ -0,0 +1,21 @@
+# WhaTap Output Plugin
+
+This plugin writes to the WhaTap(https://www.whatap.io) APM(via TCP).
+
+```toml
+  ## You can create a project on the WhaTap site(https://www.whatap.io) 
+  ## to get license, project code and server IP information.
+
+  ## WhaTap license. Required
+  #license = "xxxx-xxxx-xxxx"
+
+  ## WhaTap project code. Required
+  #pcode = 1111
+
+  ## WhaTap server IP. Required
+  ## Put multiple IPs. ["tcp://1.1.1.1:6600","tcp://2.2.2.2:6600"]
+  #servers = ["tcp://1.1.1.1:6600","tcp://2.2.2.2:6600"]

```

Comment:
Please don't comment required options.
```suggestion
  ## WhaTap license. Required
  license = "xxxx-xxxx-xxxx"

  ## WhaTap project code. Required
  pcode = 1111

  ## WhaTap server IP. Required
  ## Put multiple IPs. ["tcp://1.1.1.1:6600","tcp://2.2.2.2:6600"]
  servers = ["tcp://1.1.1.1:6600","tcp://2.2.2.2:6600"]

```
and maybe you should use `project` instead of `pcode`?

Reply:
Edited and applied. We ask for confirmation.